docs/maya/muscle.md

```diff
@@ -61,8 +61,10 @@ To create an AdnMuscle, follow these steps:
 #### Stiffness Settings
 | Name | Type | Default | Animatable | Description |
 | :--- | :--- | :------ | :--------- | :---------- |
-| **Use Custom Stiffness** | Boolean | False          | ✓ | Toggles the use of a custom stiffness value. If custom stiffness is used, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead. |
-| **Stiffness**            | Float   | 10<sup>5</sup> | ✓ | Sets the custom stiffness value. Its value must be greater than 0.0. |
+| **Use Custom Stiffness**                  | Boolean | False          | ✓ | Toggles the use of a custom stiffness value. If custom stiffness is used, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead. |
+| **Stiffness**                             | Float   | 10<sup>5</sup> | ✓ | Sets the custom stiffness value. Its value must be greater than 0.0. |
+| **Override Shape Preservation Stiffness** | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
+| **Shape Preserve Stiffness**              | Float   | 1000.0 | ✓ | Sets the override for the shape preservation stiffness to control the amount of shape preservation to be applied. |
```

Comment:
May you add some horizontal spaces to align the columns? This comment applies to all deformers.

Also I like more the following suggestion, the current description feels like we are describing weights.

```suggestion
| **Shape Preserve Stiffness** | Float | 1000.0 | ✓ | Sets the stiffness shape preservation override value. Its value must be greater than 0.0 |
```

Reply:
Done!